examples/Console/TestJaegerExporter.cs

```diff
@@ -52,10 +53,16 @@ internal static object Run(string host, int port)
 
         internal static object RunWithActivity(string host, int port)
         {
+            List<KeyValuePair<string, object>> resAttribs = new List<KeyValuePair<string, object>>()
```

Comment:
After reading this whole thing again, I think we can leave this example as is. The readme says `OpenTelemetry traces into the Jaeger model following the [OpenTelemetry specification]`, which would imply that Resources are converted as well.

Reply:
Reverting changes